CHANGES.rst

```diff
@@ -117,6 +117,8 @@ astropy.units
 astropy.utils
 ^^^^^^^^^^^^^
 
+- Fix case where ``None`` could be used in a numerical computation. [#10126]
```

Comment:
The entry might be moved depending on the milestone set. FYI.

Reply:
No problem!